src/views/emails/purchase-receipt/body.php

```diff
@@ -35,6 +35,8 @@
 
 $this->template( 'template-parts/body/order/order-total' );
 
+$this->template( 'template-parts/body/order/order-gateway-data' );
+
```

Comment:
```suggestion
```

This gateway order ID is meant for the Admin, the transaction ID for the end-users who bought the tickets can be different from this. 

This is how the end user sees the transaction in PayPal: https://share.cleanshot.com/KVmJZVpx
This is how it shows in our admin area: https://share.cleanshot.com/JqvFbzQl
This is how the Admin sees the transaction details in PayPal: https://share.cleanshot.com/lkGpZnHp